src/library_sdl.js

```diff
@@ -1101,6 +1101,7 @@ var LibrarySDL = {
         audio.webAudioNode['onended'] = function() { audio['onended'](); } // For <media> element compatibility, route the onended signal to the instance.
 
         audio.webAudioPannerNode = SDL.audioContext['createPanner']();
+        audio.webAudioPannerNode["setPosition"](0, 0, -.5);
```

Comment:
please add a comment explaining why this is needed.

also, why `-.5` and not `0` for the last one?